test/main.js

```diff
@@ -73,6 +74,35 @@ describe('gulp-istanbul', function () {
       this.stream.write(srcFile);
       this.stream.end();
     });
+
+    it('adds covStub to untested files', function (done) {
+      var COV_VAR = 'untestedCovVar';
+      var covStubs = {};
+      gulp.src([ 'test/fixtures/lib/*.js' ])
+        .pipe(istanbul({
+            coverageVariable: COV_VAR,
+            includeUntested: true
+        }))
+        .pipe(es.through(function (file) {
+          covStubs[file.path] = file.covStub;
+          this.push(file);
+        }))
+        .on('end', function () {
+          var filePaths = Object.keys(covStubs);
+          assert(filePaths.length === 2, '2 files with covStub');
+          filePaths.forEach(function (filePath) {
+            assert(covStubs[filePath].path === filePath, 'covStub has path');
+            assert(covStubs[filePath].hasOwnProperty('s'), 'covStub has s');
```

Comment:
We don't actually care about the content of the covStub property, that's an external implementation details. As soon as the module feature changes our test will fail - that's not what we want.